src/modules.js

```diff
@@ -247,7 +247,14 @@ var LibraryManager = {
             error('Function ' + x + ' aliases to target function ' + target + ', but neither the alias or the target provide a signature. Please add a ' + target + "__sig: 'vifj...' annotation or a " + x + "__sig: 'vifj...' annotation to describe the type of function forwarding that is needed!");
           }
 
+          if (typeof lib[target] !== 'function') {
+            error(`no alias found for ${x}`);
+          }
+
           var argCount = sig.length - 1;
+          if (argCount !== lib[target].length) {
+            error(`incorrect number of arguments in signature of ${x} (declared: ${argCount}, expected: ${lib[target].length})`);
```

Comment:
I guess you verified that this works?   I guess writing a test is tricky... is it worth it?

Reply:
Yes, I verified manually, and not sure a test is worth it.